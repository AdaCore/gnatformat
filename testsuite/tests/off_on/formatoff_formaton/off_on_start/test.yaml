description: |4
    Test the '--!format off' / '--!format on' at the start of the source
driver: gnatformat
args: ["--pipe", "test.adb"]
