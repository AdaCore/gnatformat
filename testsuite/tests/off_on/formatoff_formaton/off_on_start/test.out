--!format off
procedure Test is
   A :
   constant B
   := C;
   --!format on
begin
   null;
end Test;

